pkg/configuration/configuration.go

```diff
@@ -111,6 +111,9 @@ type TemplateRepositoryManifest struct {
 
 	// Arguments are a declaration of arguments to the template generator
 	Arguments map[string]Argument `yaml:"arguments,omitempty"`
+
+	// DirReplacements is a list of directory name replacement templates to render
```

Comment:
**thought**: It's probably better to make this something like `RenderOptions.DirReplacements` to make it more obvious that it's mutating the render behaviour.

Reply:
talked offline, but it feels better to me as a top class notion since it feels like more than a render option